### PR TITLE
[Doc] Use Lance with tf.data 

### PR DESCRIPTION
**Pull request by eddyxu**

### CODE REVIEW

File: docs/integrations/tf.rst
Comment (wjones127) [inline docs/integrations/tf.rst:1]: For SEO purposes, perhaps it's better to name the file `tensorflow.rst`? Also just for clarity in the directory tree.

**Review by wjones127 — APPROVED** [review]

**Comment by wjones127** [comment]
This integration looks pretty cool. Worth a blog post when we release :)

File: docs/integrations/tf.rst
Comment (eddyxu) [reply docs/integrations/tf.rst:1]: Thats a good point.

File: docs/integrations/tensorflow.rst [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+.. code-block:: python
+
+    batch_size = 256
+    ds = lance.tf.data.from_lance(
+        "s3://my-bucket/my-dataset",
+        columns=["image", "label"],
+        batch_size=batch_size,
+        output_signature={
+            "text": tf.TensorSpec(shape=(), dtype=tf.string),
```
Comment (chebbyChefNEQ) [inline docs/integrations/tensorflow.rst:50]: The column uses "image", but the specification here is "text".

File: docs/integrations/tensorflow.rst [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            "labels": tf.RaggedTensorSpec(
```
Comment (chebbyChefNEQ) [inline docs/integrations/tensorflow.rst:51]: label? Instead of labels

**Review by chebbyChefNEQ — APPROVED** [review]
Two nits

File: docs/integrations/tensorflow.rst (hunk 1 above)
Comment (eddyxu) [reply docs/integrations/tensorflow.rst]: done

File: docs/integrations/tensorflow.rst (hunk 2 above)
Comment (eddyxu) [reply docs/integrations/tensorflow.rst:51]: done

**Pull request merged**